src/app.js

```diff
@@ -165,7 +165,20 @@ app.on('before-quit', () => {
 })
 
 function addCustomCss(windowElement) {
+  const cssBaseFile = 'style.css'
```

Comment:
Why is this a variable?

Reply:
to have all the css not hardcoded in the fs.readFileSync.
Maybe later on it would be more appropriate to better abstract this part.